src/solver/td3.ml

```diff
@@ -252,6 +253,23 @@ module Base =
       let rho_write = data.rho_write in
       let dep = data.dep in
 
+      let choose_wpoint_select: string -> (module SideWPointSelect.S) = fun conf -> match conf with
+        | "always" -> (module SideWPointSelect.Always)
+        | "never" -> (module SideWPointSelect.Never)
+        | "sides-local" -> (module SideWPointSelect.SidesLocal)
+        | "sides" -> (module SideWPointSelect.Sides)
+        | "sides-pp" -> (module SideWPointSelect.SidesPP)
+        | "unstable-self" -> (module SideWPointSelect.UnstableSelf)
+        | "unstable-called" -> (module SideWPointSelect.UnstableCalled)
+        | "cycle" -> (module SideWPointSelect.Cycle)
+        | _ -> failwith ("Unknown value '" ^ conf ^ "' for option solvers.td3.side_widen!")
```

Comment:
This switching could also be moved to `SideWPointSelect` where these are actually defined.